mapie/tests/test_classification.py

```diff
@@ -334,229 +332,36 @@ def test_initialized() -> None:
-    cumclf.fit(cumclf.X_calib, cumclf.y_calib)
-    mapie = MapieClassifier(
-        cumclf,
-        method="cumulated_score",
-        cv="prefit",
-        random_state=42
-    )
-    mapie.fit(cumclf.X_calib, cumclf.y_calib, image_input=True,)
-    with pytest.raises(ValueError, match=r"Invalid X.*"):
-        mapie.predict(X_wrong_image)
-
-
-def test_undefined_model() -> None:
-    """
-    Test ValueError is raised if no model is specified with image input.
-    """
-    mapie = MapieClassifier()
-    with pytest.raises(ValueError, match=r"LogisticRegression's input.*"):
-        mapie.fit(X_good_image, y_toy_image, image_input=True,)
-
-
-@pytest.mark.parametrize("method", WRONG_METHODS)
-def test_method_error_in_fit(monkeypatch: Any, method: str) -> None:
-    """Test else condition for the method in .fit"""
-    monkeypatch.setattr(
-        MapieClassifier, "_check_parameters", do_nothing
-    )
-    mapie = MapieClassifier(method=method)
-    with pytest.raises(ValueError, match=r".*Invalid method.*"):
-        mapie.fit(X_toy, y_toy)
-
-
-@pytest.mark.parametrize("method", WRONG_METHODS)
-@pytest.mark.parametrize("alpha", [0.2, [0.2, 0.3], (0.2, 0.3)])
-def test_method_error_in_predict(method: Any, alpha: float) -> None:
-    """Test else condition for the method in .predict"""
-    mapie = MapieClassifier(method='score')
-    mapie.fit(X_toy, y_toy)
-    mapie.method = method
-    with pytest.raises(ValueError, match=r".*Invalid method.*"):
-        mapie.predict(X_toy, alpha=alpha)
-
-
-@pytest.mark.parametrize("include_labels", WRONG_INCLUDE_LABELS)
-@pytest.mark.parametrize("alpha", [0.2, [0.2, 0.3], (0.2, 0.3)])
-def test_include_label_error_in_predict(
-    monkeypatch: Any, include_labels: Union[bool, str], alpha: float
-) -> None:
-    """Test else condition for include_label parameter in .predict"""
-    monkeypatch.setattr(
-        MapieClassifier,
-        "_check_include_last_label",
-        do_nothing
-    )
-    mapie = MapieClassifier(method='cumulated_score')
-    mapie.fit(X_toy, y_toy)
-    with pytest.raises(ValueError, match=r".*Invalid include.*"):
-        mapie.predict(
-            X_toy, alpha=alpha,
-            include_last_label=include_labels
-        )
-
-
-def test_none_estimator() -> None:
-    """Test that None estimator defaults to LogisticRegression."""
-    mapie = MapieClassifier(estimator=None)
-    mapie.fit(X_toy, y_toy)
-    assert isinstance(mapie.single_estimator_, LogisticRegression)
-
-
-@pytest.mark.parametrize("estimator", [0, "estimator", KFold(), ["a", "b"]])
-def test_invalid_estimator(estimator: Any) -> None:
-    """Test that invalid estimators raise errors."""
-    mapie = MapieClassifier(estimator=estimator)
-    with pytest.raises(ValueError, match=r".*Invalid estimator.*"):
-        mapie.fit(X_toy, y_toy)
+    mapie_clf = MapieClassifier()
+    assert mapie_clf.method == "score"
+    assert mapie_clf.cv == "prefit"
+    assert mapie_clf.verbose == 0
```

Comment:
To be removed as this check is in `test_common.py`.

Reply:
Done